src/include/daos_srv/pool.h

```diff
@@ -94,6 +94,7 @@ struct ds_pool_child {
 	uuid_t		spc_uuid;	/* pool UUID */
 	d_list_t	spc_cont_list;
 
+	ABT_mutex	spc_lock;
```

Comment:
A lock in a per-thread data structure? If it's used to protect other stuff rather than this per-thread data, please move it to somewhere else.